mtp_api/apps/mtp_auth/tests/utils.py

```diff
@@ -29,7 +29,7 @@ def _get_http_authorization_token_for_user(self, user, client_id=None):
             client_id = self.APPLICATION_ID_MAP[group.name]
         application = Application.objects.get(client_id=client_id)
 
-        token = get_random_string()
+        token = get_random_string(length=35)
```

Comment:
previous length was 12. given it's just for testing, i suppose it doesn't matter that it's much bigger

Reply:
Yeah, I erred on the side of caution and used the biggest length I found across the codebase...even tho, as you say this is just for testing and it shouldn't have any impact on real security or anything.